code/modules/mob/living/living_defense.dm

```diff
@@ -246,6 +255,8 @@
 					src.pinned += O
 
 /mob/living/proc/embed(obj/item/O, var/def_zone)
+	if(O.wielded)
+		return
```

Comment:
I'm not sure about this, balance wise. The only advantage hammers have over swords is that they can't embed and this would countaract it easily.

Reply:
To use this feature you need to wield a sharp weapon, so hammers can be used in one hand. This really unfair when you lose your weapon in the middle of the battle.